README.md

```diff
@@ -58,11 +58,12 @@ Starknet Foundry is under active development! Expect a lot of new features to ap
 - [x] Contract interactions testing
 - [x] Interacting with Starknet from command line
 - [x] Multicall support
-- [ ] Cheatcodes
-- [ ] Parallel tests execution
-- [ ] Performance improvements
+- [x] Cheatcodes
+- [ ] Cheatcodes parity with Foundry
```

Comment:
Not sure about this, we aren't aiming for 100% parity and this would create expectation that we do.

Reply:
How would you phrase it?